autotst/conformer/systematic.py

```diff
@@ -102,9 +108,12 @@ def find_all_combos(
 
 
 def systematic_search(conformer,
-                      delta=float(60),
-                      cistrans=True,
-                      chiral_centers=True,
+                      delta=float(120),
+                      energy_cutoff = 10.0, #kcal/mol
+                      rmsd_cutoff = 1.0, #angstroms
```

Comment:
Let's investigate this number... I understand that this is a rough optimization but I don't want to miss the lowest energy conformer.